### PR TITLE
Added PVC and PV pkg

### PR DESCRIPTION
**Pull request by zv0n**
I've added pkgs to work with Persistent Volumes and Persistent Volume Claims.

With the PV example I only guesstimated in the apply-get-delete version because in our cluster we're not allowed to create PVs directly. I hope it works, but if anyone has a cluster where they could try it out I'd be happy if you tested it out and edited the YAML in case it doesn't work :)

### CODE REVIEW

**Comment by javaducky** [comment]
Hello @zv0n! Thank you for your PR! If you would, please be sure to run `go fmt ./...` on the project to ensure format.

I'll be taking a look at this today.

Also, given this is now your second PR, I'd love to send out some swag for you! If you'd like, please send me an email at paul@k6.io with your T-shirt size and I'll get a care package sent out.

**Comment by zv0n** [comment]
I've formatted the code, I didn't know go had this command, thanks for letting me know :)

Thanks you for the T-shirt offer! I'll be sure to get in touch :)

**Comment by javaducky** [comment]
> With the PV example I only guesstimated in the apply-get-delete version because in our cluster we're not allowed to create PVs directly. I hope it works, but if anyone has a cluster where they could try it out I'd be happy if you tested it out and edited the YAML in case it doesn't work :)

Running k3d locally, I was able to confirm PV creation and deletion. I did have to modify the YAML function as follows:
```javascript
function getPVYaml(name, size, storageClass) {
    return `apiVersion: v1
kind: PersistentVolume
metadata:
  name: ` + name + `
spec:
  capacity:
    storage: ` + size + `
  accessModes:
  - ReadWriteMany
  persistentVolumeReclaimPolicy: Retain
  storageClassName: ` + storageClass + `
  hostPath:
    path: "/tmp/k3dvol"`
}
```
tl;dr ... I added the `hostPath` section.

**Review by javaducky — APPROVED** [review]
LGTM! Confirmed operation with minor tweaks to example YAML definitions for local environment.
